modules/command-engine/core/src/test/scala/surge/internal/domain/SurgeMessagePipelineSpec.scala

```diff
@@ -132,9 +132,10 @@ class SurgeMessagePipelineSpec
   }
 
   override def afterAll(): Unit = {
-    pipeline.controllable.stop().futureValue shouldBe an[Ack]
-    TestKit.shutdownActorSystem(system, verifySystemShutdown = true)
     EmbeddedKafka.stop()
+    pipeline.controllable.stop().futureValue shouldBe an[Ack]
```

Comment:
I think there's a race condition here where if Kafka shuts down before the engine it doesn't always stop cleanly in a reasonable amount of time which is why `verifySystemShutdown = true` wasn't quite working originally, not sure about why its flaky now though

Reply:
Also not sure why it's flaky. I've tried to revert changes and now seeing a bunch of errors.
https://github.com/UltimateSoftware/surge/pull/189/commits/d9307fe5c65ff906aaab79af30046883ef7384a2